RecoEcal/EgammaCoreTools/plugins/EcalClusterEnergyCorrectionObjectSpecific.cc

```diff
@@ -335,8 +335,6 @@ float EcalClusterEnergyCorrectionObjectSpecific::fEt(float ET, int algorithm) co
     par2 = (params_->params())[172];
     par3 = (params_->params())[173];
     par4 = (params_->params())[174];
-    par5 = (params_->params())[175];//should be 0 (not used)
-    par6 = (params_->params())[176];//should be 0 (not used)
```

Comment:
some comments should be left here to explain the discontinuity in the indexing.
Maybe just comment out these lines instead of removing.